kalite/distributed/urls.py

```diff
@@ -101,6 +101,9 @@
     url(r'^store/', include(kalite.store.urls)),
     # the following pattern is a catch-all, so keep it last:
 
+    # Let user to create the super user from browser
+    url(r'^create_superuser_from_browser/$', 'create_superuser'),
```

Comment:
Maybe something like `api/django_user`? In general it seems to be something like `api/<resource_name>` or `api/<app>/<resource_name>`.